cmd/containerd/command/service_windows.go

```diff
@@ -337,6 +337,13 @@ func initPanicFile(path string) error {
 	// and replace it.
 	if st.Size() > 0 {
 		panicFile.Close()
+
+		if content, err := ioutil.ReadFile(path); err != nil {
```

Comment:
Does it make sense to Close then ReadFile rather than just reading from the opened file?

Also, while we're at it, can we do the Stat Size check and this new behavior before the OpenFile in the first place?